packages/fxa-auth-server/lib/routes/password.ts

```diff
@@ -111,89 +205,31 @@ module.exports = function (
-            oldAuthPW,
-            emailRecord.authSalt,
-            emailRecord.verifierVersion
-          );
-
-          const match = await signinUtils.checkPassword(
-            emailRecord,
-            password,
-            request
-          );
-
-          if (!match) {
-            throw error.incorrectPassword(emailRecord.email, form.email);
-          }
-
-          if (password.clientVersion === 1) {
-            const unwrappedKb = await password.unwrap(emailRecord.wrapWrapKb);
-            keyFetchToken = await db.createKeyFetchToken({
-              uid: emailRecord.uid,
-              kA: emailRecord.kA,
-              wrapKb: unwrappedKb,
-              emailVerified: emailRecord.emailVerified,
-              tokenVerificationId: sessionToken?.tokenVerificationId,
-            });
-          }
-
-          if (password.clientVersion === 2) {
-            const unwrappedKb = await password.unwrap(
-              emailRecord.wrapWrapKbVersion2
-            );
-            keyFetchToken2 = await db.createKeyFetchToken({
-              uid: emailRecord.uid,
-              kA: emailRecord.kA,
-              wrapKb: unwrappedKb,
-              emailVerified: emailRecord.emailVerified,
-              tokenVerificationId: sessionToken?.tokenVerificationId,
-            });
-          }
-
-          passwordChangeToken = await db.createPasswordChangeToken({
-            uid: emailRecord.uid,
-          });
-        } catch (err) {
-          if (err.errno === error.ERRNO.ACCOUNT_UNKNOWN) {
-            customs.flag(request.app.clientAddress, {
-              email: form.email,
-              errno: err.errno,
-            });
-
-            if (customs.v2Enabled()) {
-              await customs.check(request, email, 'passwordChangeStartFailed');
-            }
-          }
-          throw err;
-        }
+        return await handlePasswordChangeStart(request, uid, account);
+      },
+    },
+    {
+      method: 'POST',
+      path: '/password/change/start/jwt',
+      options: {
+        ...PASSWORD_DOCS.PASSWORD_CHANGE_START_POST,
+        auth: {
+          strategy: 'mfa',
+          scope: ['mfa:password'],
+          payload: false,
+        },
+        validate: {
+          payload: isA.object({
+            email: validators.email().required().description(DESCRIPTION.email),
+            oldAuthPW: validators.authPW.description(DESCRIPTION.authPW),
+          }),
+        },
+      },
+      handler: async function (request: AuthRequest) {
+        const { uid } = request.auth.credentials as { uid: string };
+        const account = await db.account(uid);
 
-        return {
-          keyFetchToken: keyFetchToken?.data,
-          keyFetchToken2: keyFetchToken2?.data,
-          passwordChangeToken: passwordChangeToken?.data,
-          verified:
-            keyFetchToken?.emailVerified || keyFetchToken2?.emailVerified,
-        };
+        return await handlePasswordChangeStart(request, uid, account);
```

Comment:
We should be able to add the same check that exists on the session token version. e.g.

```
const { uid, emailVerified } = request.auth.credentials as { uid: string };

if (!emailVerified) {
    statsd.increment('passwordChange.start.emailNotVerified');
    throw error.unverifiedSession();
}
```